breaking_changes.md

```diff
@@ -0,0 +1,9 @@
+# mu_feature_config Breaking Changes

```

Comment:
I'm fine tracking this a different way, just one idea on how to show users that a breaking change has happened, without digging through PRs. Perhaps this could be automated and pull any PRs that get committed with the breaking change tag, grab their description and integration instructions, toss them in this file with the date commit hash, but for now... :).